traffic/core/traffic.py

```diff
@@ -743,6 +753,57 @@ def geoencode(self, *args, **kwargs):
         """
         raise NotImplementedError
 
+    def geoencode_groundtraffic(
```

Comment:
I will reject this part for this PR, but keep the suggestion for another increment